spec/beaker-answers/beaker-answers_spec.rb

```diff
@@ -24,548 +26,564 @@
     expect(opts[:answers][:q_puppet_cloud_install]).to be == 'n'
     hosts.each do |host|
       if @answers[host.name]
-        expect( @answers[host.name][:q_puppet_cloud_install] ).to be == 'n'
+        expect(@answers[host.name][:q_puppet_cloud_install]).to be == 'n'
       end
     end
   end
 
   context 'when we are upgrading to a version > 3.8' do
-    supported_general_upgrade_versions = [ '2015.1.0',
-                                           '2016.1.0',
-                                           '2016.2.1']
+    supported_general_upgrade_versions = ['2015.1.0',
+                                          '2016.1.0',
+                                          '2016.2.1',]
     supported_general_upgrade_versions.each do |version|
-      it "still creates the full install answers" do
+      it 'still creates the full install answers' do
         @ver = version
         options[:type] = :upgrade
-        expect( answers ).to be_a_kind_of BeakerAnswers::Answers
-        expect( answers ).to_not be_a_kind_of BeakerAnswers::Upgrade
+        expect(answers).to be_a_kind_of BeakerAnswers::Answers
+        expect(answers).not_to be_a_kind_of BeakerAnswers::Upgrade
```

Comment:
Let's not switch this arbitrarily.  If we want to update rspec syntax, let's handle that separately.

Reply:
This one is called [Rspec/NotToNot](http://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/NotToNot) and I've disabled it for now